Conceptpower+Spring/src/test/java/edu/asu/conceptpower/app/core/impl/RequestsManagerTest.java

```diff
@@ -49,9 +78,11 @@ public void test_getReview_success() {
         reviewRequest.setConceptId("WID-10126926-N-05-Einstein");
         reviewRequest.setRequest("Testing Method2");
         
-        Mockito.when(dbClient.getReviewRequestForConcept("WID-10126926-N-05-Einstein")).thenReturn(reviewRequest);
+        List<ReviewRequest> response = new ArrayList<>();
+        response.add(reviewRequest);
+        Mockito.when(dbClient.getAllReviews("WID-10126926-N-05-Einstein")).thenReturn(response);
```

Comment:
is this necessary? you already set up returning a response list in the before method, dont' you?

Reply:
?